### DISCUSSION TITLE
Projects using MIR

### DISCUSSION THREAD

**Discussion by dibyendumajumdar** [vnmakarov/mir#173] [Show and tell]
Hi, staring a thread on projects that are using MIR. Please post your project here!

**Comment by dibyendumajumdar** [comment]
I will start with my own project: Ravi

https://github.com/dibyendumajumdar/ravi

Ravi is a Lua dialect that uses MIR project as the JIT backend.

**Comment by sletz** [comment]
[Faust](https://faust.grame.fr) is a functional programming language for sound synthesis and audio processing. It contains a mixed interpreter/MIR backend as described here: https://github.com/vnmakarov/mir/issues/54

**Comment by Itay2805** [comment]
[tinydotnet](https://github.com/Itay2805/tinydotnet) is a small and lightweight dotnet runtime implementation with the hope to eventually embed it inside a kernel.

**Reply by vnmakarov** [reply]
Thank you for informing me about your project and using MIR for it.  I wish you a successful outcome.  On my side I am planning to work on improving MIR in the future.
